### ISSUE TITLE
Filters cannot be accessed with the keyboard

### ISSUE THREAD

**Issue by rlucke** [opencast/opencast-admin-interface#624]
![Screenshot 2024-06-05 123632](https://github.com/opencast/opencast-admin-interface/assets/1741658/913f5e4f-c6b4-4f56-a19f-98f3a3c4c3eb)

Filters in the header area cannot be accessed with the keyboard
